package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -67,9 +67,9 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
         //  m_usdScene.Stage.SetTimeCodesPerSecond(1);
         //}
 
-        // TODO: How does one extract the time mode (frames or seconds) from the Timeline?
-        Clip.UsdScene.Stage.SetFramesPerSecond(30);
-        Clip.UsdScene.Stage.SetTimeCodesPerSecond(1);
+        Clip.UsdScene.FrameRate = 60;
+        Clip.UsdScene.Stage.SetFramesPerSecond(60);
+        Clip.UsdScene.Stage.SetInterpolationType(pxr.UsdInterpolationType.UsdInterpolationTypeLinear);
```

Comment:
Please add a comment here and note that:

```
Regardless of the actual sampling rate (e.g. Timeline playback speed), we are converting
the timecode from seconds to frames with a sampling rate of 60 FPS. This has the nice quality
of adding additional numerical stability.

In the event that the timeline is not configured for 60 FPS playback, we rely on USD's linear
interpolation mode to up-sample to 60 FPS.
```

Reply:
Added.

Note that I'm not 100% sure that this actually adds numerical stability; the timeline is just too "off beat" to produce exact, int-aligned numbers here. You can see some exported timings in the screenshot in my PR. For exact int-aligned frame numbers (and preventing that initial gap) more work would be needed.

Reply:
Ah, I see, well in theory it should enable us to put key frames at exact integer frame offsets (e.g. for offline export), but yeah, it's probably not going to help without putting some effort on making that more consistent.